app/controllers/api/mixins/service_templates.rb

```diff
@@ -17,6 +19,14 @@ def order_service_template(id, data, scheduled_time = nil)
       def service_template_ident(st)
         "Service Template id:#{st.id} name:'#{st.name}'"
       end
+
+      def api_request_allowed?
+        standalone_api_request? ? !Settings.product.deny_api_service_ordering : true
+      end
+
+      def standalone_api_request?
+        request.authorization.present? && request.authorization.to_s.split(" ", 2).first.downcase == "basic"
```

Comment:
`request.authorization.try(:starts_with?, "basic")`

Reply:
Also, this will break if we allow other authentication types to the api

Reply:
Agreed, but do you know of another way to distinguish standalone API calls versus API calls made from the UI?

Reply:
I can't think of anything at the moment

Reply:
maybe we can leverage the token manager namespace, api, ws,  (might need to add param or separate api/ui as 2 namespaces).

Reply:
Also, I pulled the "basic" check from here: https://github.com/rails/rails/blob/fc5dd0b85189811062c85520fd70de8389b55aeb/actionpack/lib/action_controller/metal/http_authentication.rb#L104.

I suppose we could do `.try(:downcase).try(:starts_with?, "basic")`? Not sure how else to handle the fact that it could be "Basic", "BASIC", or any other various capitalizations.